manifests/jail.pp

```diff
@@ -14,6 +14,7 @@
   Integer $bantime = $fail2ban::bantime,
   Optional[String] $port = undef,
   Optional[String] $backend = undef,
+  Optional[Array[Stdlib::IP::Address]] $ignoreip = [],
```

Comment:
you can remove the `Optional[]`

Reply:
Heh, I was wondering if this was still necessary. Removing it.